frontend/src/components/Wind.jsx

```diff
@@ -115,13 +111,13 @@ const WindSettings = ({
 
     const addNewWindBlock = () => {
         const newWindBlock = {
-          windType: selectedWindType,
-          windDirection: envConf.Wind.Direction,
-          windVelocity: envConf.Wind.Force,
-          fluctuationPercentage: selectedWindType === 'Turbulent Wind' ? fluctuationPercentage : 0,
+          windType: "Constant Wind",
+          windDirection: "N",
```

Comment:
@syedomair0 I remember default WindDirection was "NE" and default windVelocity was 1 earlier. Can you just replace them?